src/test/java/core/basesyntax/strategy/handler/impl/ReturnHandlerTest.java

```diff
@@ -0,0 +1,83 @@
+package core.basesyntax.strategy.handler.impl;
+
+import static org.junit.jupiter.api.Assertions.assertSame;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.handler.OperationHandler;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class ReturnHandlerTest {
+
+    private static final String APPLE = "apple";
+    private static final int VALID_FRUIT_QUANTITY = 5;
+    private static final String BANANA = "banana";
+    private static final int NEGATIVE_FRUIT_QUANTITY = -7;
+    private static final int ZERO_FRUIT_QUANTITY = 0;
+    private static final int ADD_FRUIT_QUANTITY = 7;
+    private static OperationHandler handler;
+
+    @BeforeAll
+    public static void beforeAll() {
+        handler = new ReturnHandler();
+    }
+
+    @BeforeEach
+    public void setUp() {
+        Storage.FRUITS.put(BANANA, VALID_FRUIT_QUANTITY);
```

Comment:
It's not necessary to setUp fruits before each case, better use special put for every test case, pbt it in cases, where you realy need it for test, because it's a little bit weird, when you put some value before each and clean storage in some test's, because you don't need to have values in storage in that case